cm.py

```diff
@@ -125,11 +125,25 @@ def parcel(self, product=None, version=None, stage=None):
         if not product and not version and not stage:
             raise Exception('A product and/or version and/or stage must '
                             'be specified to select a parcel.')
-        return next(parcel
+        return next((parcel
                     for parcel in self.parcels
                     if (not product or parcel.product == product)
                     and (not version or parcel.version == version)
-                    and (not stage or parcel.stage == stage))
+                    and (not stage or parcel.stage == stage)), None)
+
+    def wait_for_parcel_availability(self, product, version=None, stage=None):
```

Comment:
Can we call this `wait_for_parcel_stage` since then it could also be used, for example, to wait for a parcel to become undistributed or something like that?

Reply:
Done.